test/langtools/tools/javac/reflect/LocalClassTest.java

```diff
@@ -71,11 +95,26 @@ void m() { }
     @CodeReflection
     @IR("""
             func @"testLocalCaptureParam" (%0 : java.type:"LocalClassTest", %1 : java.type:"java.lang.String")java.type:"java.lang.String" -> {
-                %2 : Var<java.type:"java.lang.String"> = var %1 @"s";
-                %3 : java.type:"java.lang.String" = var.load %2;
-                %4 : java.type:"LocalClassTest::$2Foo" = new %0 %3 @java.ref:"LocalClassTest::$2Foo::(LocalClassTest, java.lang.String)";
-                %5 : java.type:"java.lang.String" = invoke %4 @java.ref:"LocalClassTest::$2Foo::m():java.lang.String";
-                return %5;
+                  %2 : Var<java.type:"java.lang.String"> = var %1 @"s";
+                  class.dec ()java.type:"void" -> {
+                      %3 : Var<java.type:"LocalClassTest"> = var;
+                      %4 : Var<java.type:"java.lang.String"> = var;
+                      func @"<init>" (%5 : java.type:"LocalClassTest", %6 : java.type:"java.lang.String")java.type:"void" -> {
+                          invoke @java.ref:"java.lang.Object::()" @invoke.kind="SUPER";
+                          var.store %3 %5;
+                          var.store %4 %6;
+                          return;
+                      };
+                      func @"m" (%7 : java.type:"LocalClassTest::$2Foo")java.type:"java.lang.String" -> {
+                          %8 : java.type:"java.lang.String" = var.load %4;
+                          return %8;
+                      };
+                      yield;
+                  };
+                  %9 : java.type:"java.lang.String" = var.load %2;
+                  %10 : java.type:"LocalClassTest::$2Foo" = new %0 %9 @java.ref:"LocalClassTest::$2Foo::(LocalClassTest, java.lang.String)";
```

Comment:
How do we link the name `LocalClassTest::$2Foo` back to the new op?

Reply:
Should the new op refer to the local class ?